packages/cli/src/__tests__/fix.ts

```diff
@@ -70,6 +70,59 @@ test("set main and module field", async () => {
   `);
 });
 
+test("set exports field when opt-in", async () => {
+  let tmpPath = f.copy("package-exports");
+
+  await modifyPkg(tmpPath, (json) => {
+    json.exports = { ".": { browser: {}, worker: {} } };
+  });
+
+  await fix(tmpPath);
+
+  let pkg = await getPkg(tmpPath);
+
+  expect(pkg).toMatchInlineSnapshot(`
+    Object {
+      "exports": Object {
+        ".": Object {
+          "browser": Object {
+            "default": "./dist/package-exports.browser.cjs.js",
+            "module": "./dist/package-exports.browser.esm.dev.js",
+            "production": Object {
+              "default": "./dist/package-exports.browser.cjs.prod.js",
+              "module": "./dist/package-exports.browser.esm.prod.js",
+            },
+          },
+          "default": "./dist/package-exports.cjs.js",
```

Comment:
It's essential that default is always the last condition otherwise the other ones won't get matched

Reply:
You are right - but those snapshots have tricked us. Jest is sorting properties on objects 🤦 so it only looked like it was incorrect but in reality this was fine.

I've prepared a workaround to avoid sorting of those properties in snapshots: https://github.com/nicksrandall/preconstruct/pull/1

Reply:
I have added a note to this specific test describing that the keys in the snapshot are technically out of order because jest is sorting them. I've also added some additional assertions to the test to enforce that conditions are indeed created in the correct order.